tests/config.example.sh

```diff
@@ -2,4 +2,6 @@
 
 AWS_ACCESS_KEY_ID=""
 AWS_SECRET_ACCESS_KEY=""
-GH_ACTION_WORKFLOW_PATH="./.cluster.dev/gh-5.yaml" # Relative path from project root
+# Timeout in seconds.
+ACTION_TIMEOUT="600"
```

Comment:
I recommend add more info for what propose this var needed
Like this
```bash
GH_ACTION_RUN_TIMEOUT="600"
```
or this
```bash
ACTION_TIMEOUT="600" # After timeout Github Action will be stopped
```

Reply:
Agree with that proposal